src/game.cpp

```diff
@@ -4214,7 +4224,7 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats,
 	if (draw_control->range_all) {
 		runData->fog_range = 100000 * BS;
 	} else {
-		runData->fog_range = 0.9 * draw_control->wanted_range * BS;
+		runData->fog_range = draw_control->wanted_range * BS;
```

Comment:
While I do agree, some previous PRs which I had taken care to split into different commits were squashed on merge, so I'm a bit confused as to what is expected.

I'll be more than happy to split this PR into two commits.